File: app/src/main/java/com/liushu/crazyandroid/ui/stage01/chapter02/activity/Demo020103Activity.java
package com.liushu.crazyandroid.ui.stage01.chapter02.activity;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import com.jaydenxiao.common.base.BaseActivity;
import com.liushu.crazyandroid.R;

import butterknife.Bind;

public class Demo020103Activity extends BaseActivity {
    @Bind(R.id.iv_back)
    ImageView mIvBack;
    @Bind(R.id.tv_title_name)
    TextView mTvTitleName;

    @Override
    public int getLayoutId() {
        return R.layout.activity_demo020103;
    }

    @Override
    public void initPresenter() {

    }

    @Override
    public void initView() {
        mTvTitleName.setText("跟随手指的小球");
        mIvBack.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                finish();
            }
        });

    }
}

File: app/src/main/java/com/liushu/crazyandroid/ui/stage01/chapter02/activity/Demo020103Activity.kt
package com.liushu.crazyandroid.ui.stage01.chapter02.activity

import com.jaydenxiao.common.base.BaseActivity
import com.liushu.crazyandroid.R
import kotlinx.android.synthetic.main.head_title_layout.*

class Demo020103Activity : BaseActivity() {


    override val layoutId: Int
        get() = R.layout.activity_demo020103


    override fun initView() {
        tv_title_name!!.text = "跟随手指的小球"
        iv_back!!.setOnClickListener { finish() }

    }
}
